test-clients/src/main/java/com/hedera/services/bdd/spec/HapiApiSpec.java

```diff
@@ -214,12 +233,17 @@ private void exec(List<HapiSpecOperation> ops) {
 			return;
 		}
 
+		List<HapiSpecOperation> creationOps = entities.requiredCreations();
```

Comment:
A silly comment here.  Since most of the develop side unit and regression test won't involve the persistence of entities,  it would be helpful if the EntityManager related operations can be made configurable in future iterations.

Reply:
Agreed, I pushed a change to make entity-related calls in the `HapiApiSpec` lifecycles **only if** the `persistentEntities.dir.path` property is explicitly set.